analyzere/utils.py

```diff
@@ -92,3 +92,17 @@ def vectorize(values):
     if isinstance(values, list):
         return ','.join(str(v) for v in values)
     return values
+
+
+def vectorize_range(values):
```

Comment:
this string manipulation is for the purposes of url encoding?  If so, a comment to that effect would be good.